ecl/hql/hqlexpr.hpp

```diff
@@ -789,7 +789,8 @@ enum ExprPropKind
     EPunadorned,
     EPlocationIndependent,
     EPmeta,
-    EPmax
+    EPmax,
```

Comment:
EPmax needs to be the last one - we often use an enumeration like this at the end of a list to indicate the number of elements.